src/pages/demo.tsx

```diff
@@ -144,13 +143,13 @@ const MarinaExample: React.FC<Props> = () => {
               Send LBTC
             </button>
             {
-              tx.length > 0 &&
+              txHash.length > 0 &&
               <>
                 <p>
-                  {tx.substring(0, 20) + "..." + tx.substring((tx.length - 20), tx.length)}
+                  {txHash.substring(0, 20) + "..." + txHash.substring((txHash.length - 20), txHash.length)}
```

Comment:
maybe now is fine to show it entirely since much more shorter

Reply:
Yes, and leave a safeguard in CSS in case it overflows on smaller displays.
On it.

Reply:
Forget it, is a demo 😄 
Doesn't need the CSS safeguard.

Reply:
Commit 1fd37ba fixes this.